README.md

```diff
@@ -1,37 +1,69 @@
 
-The UID 2 Project is subject to Tech Lab IPR’s Policy and is managed by the IAB Tech Lab Addressability Working Group and Privacy & Rearc Commit Group. Please review the governance rules [here](https://github.com/IABTechLab/uid2-core/blob/master/Software%20Development%20and%20Release%20Procedures.md)
+The UID 2 Project is subject to Tech Lab IPR’s Policy and is managed by the IAB Tech Lab Addressability Working Group and Privacy & Rearc Commit Group. Please review the governance rules [here](https://github.com/IABTechLab/uid2-core/blob/master/Software%20Development%20and%20Release%20Procedures.md).
 
-## Build & Package
+# Requirements
 
-```
-    mvn package
-```
+* Java 1.8 or later
 
-## Install
 
-1. add reference to uid2-client-2.0.0-jar-with-dependencies.jar generated from build & package setup in your project
+## Installation
 
-2. install the library locally and reference via maven
+Add this dependency to your project's POM:
 
-```
-    mvn install:install-file -Dfile="./target/uid2-client-2.0.0-jar-with-dependencies.jar" -DgroupId="com.uid2" -DartifactId="uid2-client" -Dpackaging=jar -Dversion="2.0.0"
-```
-
-this will install the jar into your local .m2 repository, you can set your desired target repo with `-DlocalRepositoryPath=path-to-specific-local-repo`
-
-then on your maven project's pom.xml, add
 
 ```
         <dependency>
             <groupId>com.uid2</groupId>
             <artifactId>uid2-client</artifactId>
-            <version>2.0.0</version>
+            <version>4.0.0</version>
```

Comment:
Why the jump to 4?

Reply:
3.x was testing the Maven Central release but with no changes from 2.x: https://search.maven.org/search?q=a:uid2-client
4.x are these publisher sdk changes

Could be argued that we don't need to jump to 4 though